Gemfile

```diff
@@ -85,6 +85,14 @@ unless packaging?
     gem "guard", require: false
     gem "guard-rspec", require: false
     gem "guard-rubocop", require: false
+    gem "jquery"
+    gem "jquery-rails"
+    gem "peek"
+    gem "peek-gc"
+    gem "peek-git"
+    gem "peek-mysql2"
+    gem "peek-performance_bar"
+    gem "peek-pg"
```

Comment:
Could you add `peek-host` ? It's kind of important considering JWT tokens will contain the hostname, and it wouldn't be the first time we have a problem because we forgot to set it properly 😄